pkg/dyninst/exprlang/testdata/deeply_nested_getmember.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "unsupported",
+  "ref": "",
+  "operation": "getmember",
+  "argument": [
+    {
+      "getmember": [
+        {
+          "getmember": [
+            {
+              "ref": "self"
+            },
+            "field1"
+          ]
+        },
+        "field2"
+      ]
+    },
+    "name"
+  ],
+  "error": ""
```

Comment:
I think this thing where ref and argument are both in the output at the same time is weird, and error should also be omitempty. Generally this output format could use another pass, but I'll leave that to you.

Reply:
I'll add back omitempty from these fields. I got rid of it because it was was messing with the require checks but I improved the logic there.